src/_locales/ja/messages.json

```
@@ -260,69 +260,69 @@
-    "wallet_balance_notice": "Notice: A small gas fee will occur for publishing.",
+    "wallet_balance_notice": "注意: 取引には少量の Gas が発生します。",
     "wallets": "仮想通貨ウォレット",
-    "untitled_wallet": "無題の財布",
+    "untitled_wallet": "無題の仮想通貨ウォレット",
     "your_message": "あなたのメッセージ...",
     "plugin_red_packet_display_name": "プラグイン: Red Packet",
-    "plugin_red_packet_select_wallet": "財布を選択",
-    "plugin_red_packet_set_as_default": "デフォルトを設定",
+    "plugin_red_packet_select_wallet": "仮想通貨ウォレットを選択",
+    "plugin_red_packet_set_as_default": "デフォルトに設定",
     "plugin_red_packet_split_mode": "分割モード",
     "plugin_red_packet_average": "平均",
     "plugin_red_packet_random": "ランダム",
     "plugin_red_packet_total_amount": "総数",
     "plugin_red_packet_amount_per_share": "毎株数量",
     "plugin_red_packet_shares": "株数",
-    "plugin_red_packet_best_wishes": "こんにちは",
-    "plugin_red_packet_not_valid": "無効入力",
-    "plugin_red_packet_send": "{{symbol}}{{amount}} {{type}} をお送り",
-    "plugin_red_packet_create_new": "新しいのを作成",
-    "plugin_red_packet_select_existing": "現存のを選択",
-    "plugin_red_packet_create_failed": "制作失敗",
-    "plugin_red_packet_attached_message": "付言",
+    "plugin_red_packet_best_wishes": " 大吉大利！",
+    "plugin_red_packet_not_valid": "無効",
+    "plugin_red_packet_send": "{{symbol}}{{amount}} {{type}} を送る",
+    "plugin_red_packet_create_new": "新規作成",
+    "plugin_red_packet_select_existing": "既存を選択",
+    "plugin_red_packet_create_failed": "作成失敗",
+    "plugin_red_packet_attached_message": "メッセージを追加",
     "plugin_red_packet_from": "送信者: {{from}}",
     "plugin_red_packet_user_status_opening": "受領中",
-    "plugin_red_packet_user_status_pending": "Pending",
-    "plugin_red_packet_description_pending": "制作中",
-    "plugin_red_packet_description_incoming": "準備完了",
-    "plugin_red_packet_description_fail": "制作失敗",
-    "plugin_red_packet_description_claimed": "もう先に受領した",
-    "plugin_red_packet_description_expired": "もう期限切れた",
-    "plugin_red_packet_description_claim_pending": "開くて",
-    "plugin_red_packet_description_empty": "もう期限切れた",
-    "plugin_red_packet_description_failover": "総数：{{total}} {{name}} / {{shares}} 株",
+    "plugin_red_packet_user_status_pending": "保留中",
+    "plugin_red_packet_description_pending": "作成中",
+    "plugin_red_packet_description_incoming": "開封可能です",
+    "plugin_red_packet_description_fail": "作成失敗",
+    "plugin_red_packet_description_claimed": "既に受け取り済みです",
+    "plugin_red_packet_description_expired": "期限切れです",
+    "plugin_red_packet_description_claim_pending": "受け取り中",
+    "plugin_red_packet_description_empty": "期限切れです",
+    "plugin_red_packet_description_failover": "合計：{{total}} {{name}} / {{shares}} 株",
     "plugin_gitcoin_display_name": "プラグイン: Gitcoin Grant",
-    "plugin_gitcoin_readme": "このサービスを利用するで、寄附金額の 5% は <fund>Gitcoin grants development fund</fund> へ寄附しています。",
+    "plugin_gitcoin_readme": "このサービスを利用するにあたり、寄付金額の 5% が <fund>Gitcoin grants development fund</fund> に寄付されます",
```

```suggestion
    "plugin_gitcoin_readme": "このサービスを利用するにあたり、寄付金額の 5% が <fund>Gitcoin grants development fund</fund> に寄付されます",
```

`Gitcoin grants development fund` not translated to Japanese?

---

I think we don't need to translate that.

Gitcoin itself don't have a Japanese version. Leave it as it

---

OK.

---

I thought it is a name of some organization. Isn't it?
If yes, remain it is fine for me.